nbs/models.kan.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #134.                self.hidden_layers = [self.input_size] + [self.hidden_size for _ in range(self.n_hidden_layers)] + [self.h]
</pre><p>I suggest that we avoid explicit for loops here.</p><p> </p><p>The second term can be simplified to <code>self.n_hidden_layer*[self.hidden_size]</code> to improve readability. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/999/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>